src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -31,9 +40,12 @@ public void lowerPiston() {
         shooterSolenoid.set(false);
     }
 
-    /** @param shooterSpeed Velocity of shooter motors in RPM */
     public void shoot(double shooterSpeed) {
-        // shooterController.setReference(shooterSpeed, ControlType.kVelocity);
-        shooterMotor1.set(shooterSpeed);
+        shooterController.setReference(2000, ControlType.kVelocity);
+        // shooterMotor1.set(shooterSpeed);
+    }
+
+    public void periodic() {
+        System.out.println(this.shooterEncoder.getVelocity());
```

Comment:
Perhaps networktables would be a better way to get this info for debugging?

Reply:
Indeed. We'll probably make this change next time. I wish there was a `tunable` for Java wpilib...